e2e/support/test_support.go

```diff
@@ -1933,3 +1933,9 @@ func GetOutputStringAsync(cmd *cobra.Command) func() string {
 		return buffer.String()
 	}
 }
+
+func IsAPIResourceInstalled(t *testing.T, client client.Client, groupVersion string, obj interface{}) bool {
```

Comment:
Can't `kubernetes.IsAPIResourceInstalled` be used?

Reply:
sure